### DISCUSSION TITLE
Units set

### DISCUSSION THREAD

**Discussion by IntePNum** [peridigm/peridigm#336] [General]
When I test the examples in the yaml and xml files.
I found that different UNIT is used, but I don't find where to set the standard of the unit of the values?

can you tell me how to distinguish the UNIT I used in my yaml/xml file?
![image](https://github.com/peridigm/peridigm/assets/108223279/1de778d1-4dfe-4cc1-940f-875cb33b2aff)

**Comment by paulordutra** [comment]
You only have to set a consistent unit system (SI or CGS, for example). Peridigm doesn't have a standard unit system.

**Reply by IntePNum** [reply]
wowww!  professor,   
thank you for your answer. I cannot find the command to set SI or CGS in the guidebook.
can you please tell me and teach me how to use the command?
best wishes,
intep,

**Reply by paulordutra** [reply]
There's not a command for this. Just check if your units for density, Young's modulus and other properties are consistent. If you use SI, use SI for all material properties. The same is valid for other systems. You can find some possible systems and units in Table 2.2 of this file:  https://elib.dlr.de/126320/1/Peridigm_Users_Guide.pdf

**Reply by IntePNum** [reply]
ok, I see, Thank you for the answer. 
you are very kind.
beste regardes.
intep.
